src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -440,6 +443,12 @@ MulticopterPositionControl::poll_subscriptions()
 	if (updated) {
 		orb_copy(ORB_ID(vehicle_trajectory_waypoint), _traj_wp_avoidance_sub, &_traj_wp_avoidance);
 	}
+
+	orb_check(_vehicle_att_sub, &updated);
```

Comment:
Comment why this is being done and perhaps only bother with the orb check and copy if you're in the relevant condition (non-finite lpos yaw?).